src/pudl/metadata/classes.py

```diff
@@ -1971,19 +1977,18 @@ def from_data_source_ids(
             output_path: PUDL_OUTPUT path.
             data_source_ids: ids of data sources currently included in Datasette
             xbrl_ids: ids of data converted XBRL data to be included in Datasette
-            extra_etl_groups: ETL groups with resources that should be included
+            exclude_intermediate_resources: exlude intermediate resources if True
         """
         # Compile a list of DataSource objects for use in the template
         data_sources = [DataSource.from_id(ds_id) for ds_id in data_source_ids]
 
         # Instantiate all possible resources in a Package:
-        pkg = Package.from_resource_ids()
-        # Grab a list of just the resources we want to output:
-        resources = [
-            res
-            for res in pkg.resources
-            if res.etl_group in data_source_ids + extra_etl_groups
-        ]
```

Comment:
I removed this filtering by source id and etl_group because all of our output table metadata was being excluded. Also, I think we want all resources to be loaded into datasette. 

[`out_eia860__yearly_emissions_control_equipment`](https://data.catalyst.coop/pudl/out_eia860__yearly_emissions_control_equipment) is the only table with metadata right now on datasette.

Reply:
I added a test in `devtools/datasette/publish.py`.

Reply:
I think this test would be more useful in the integration tests if it's easy to put it there instead. Otherwise we'll only discover that we've broken something in the nightly builds rather than in CI.

Reply:
I think this test is now being run both when we deploy and in the integration tests.